credstash.py

```diff
@@ -907,6 +911,14 @@ def get_parser():
                                   "CREDSTASH_DEFAULT_TABLE env variable, "
                                   "or if that is not set, the value "
                                   "`credential-store` will be used")
+    parsers['super'].add_argument("--endpoint_url", default=os.environ.get("DYNAMODB_ENDPOINT_URL", None),
```

Comment:
Super-small issue - could you please change this to `--endpoint-url` to be consistent with the other args?
```suggestion
    parsers['super'].add_argument("--endpoint-url", default=os.environ.get("DYNAMODB_ENDPOINT_URL", None),
```

Reply:
Done 👍